### PR TITLE
Update folder structure for custom pages

### PR DESCRIPTION
**Pull request by huijing**
This PR modifies the folder structure of our custom pages to be more in line with the recommendations from Docusaurus (see https://docusaurus.io/docs/creating-pages#routing).

There should be no discernible changes in the pages other than a few minor content updates to the meta-tag.js page (which will be left alone in this PR for now). We have separate issue (https://github.com/WICG/webmonetization/issues/322) to deal with the meta-tag.js page, and thus the folder change will take place in that PR instead.

**🎩 Top-hatting instructions (i.e. human testing that things are not broken)**
1. Pull down this branch
2. Check the following pages are rendering without error:
    - http://localhost:3000/
    - http://localhost:3000/meta-tag
    - http://localhost:3000/prob-revshare
    - http://localhost:3000/exclusive-content

### CODE REVIEW

File: src/pages/exclusive-content/components/active-view.js [hunk 1]
```diff
@@ -13,15 +12,15 @@ export function ActiveView() {
   if (view === ViewStates.Generate) {
     return (
       <>
-        <SectionHeader>Generate Exclusive Content</SectionHeader>
```
Comment (huijing) [inline src/pages/exclusive-content/components/active-view.js:16]: This component doesn't seem to be very necessary if we have chosen to standardise the baseline typographic CSS for headings.

File: src/pages/exclusive-content/components/copy-content.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export function CopyContent(props) {
   return (
     <div>
       <Tooltip title={props.message}>
-        <Button tooltip onClick={() => copyContent(props.id)}>
```
Comment (huijing) [inline src/pages/exclusive-content/components/copy-content.js:40]: The Button component does not have this prop 🧐 and it was causing console errors

File: src/pages/exclusive-content/components/generate-form.js [hunk 3]
```diff
@@ -85,9 +87,11 @@ export function GenerateForm() {
       </FormControl>
 
       <Button
+        className='content-generator__btn'
         variant='outlined'
         size='medium'
-        disabled={!pointer | !plaintext}
+        disabled={missingFields}
+        type='submit'
```
Comment (huijing) [inline src/pages/exclusive-content/components/generate-form.js:94]: This allows keyboard enter to work

File: src/pages/meta-tag.js [hunk 4]
```diff
@@ -67,14 +71,17 @@ export default function MetaTag(props) {
               setInvalidUrl('')
             }}
           />
-          <button
+          <Button
```
Comment (huijing) [inline src/pages/meta-tag.js:74]: Using the Material UI button component for consistency.
Also adding `submit` to make the keyboard enter work properly

File: src/pages/prob-revshare/components/share-input.js [hunk 5]
```diff
@@ -10,7 +10,7 @@ import { Delete } from '@mui/icons-material'
 import { validatePointer, validateWeight } from '../lib'
 
 export function ShareInput({
-  key,
```
Comment (huijing) [inline src/pages/prob-revshare/components/share-input.js:13]: `key` is a reserved React prop and cannot be passed like this

File: src/pages/prob-revshare/lib/index.js [hunk 6]
```diff
@@ -9,55 +9,50 @@ const CHART_COLORS = [
   '#297244',
   '#1C5F32',
   '#0E4C21',
-  '#003A10'
+  '#003A10',
```
Comment (huijing) [inline src/pages/prob-revshare/lib/index.js:12]: Auto-formatter hit on this file, sorry.

**Review by sublimator — APPROVED** [review]

File: src/pages/exclusive-content/components/generate-form.js [hunk 7]
```diff
@@ -85,9 +87,11 @@ export function GenerateForm() {
       </FormControl>
 
       <Button
+        className='content-generator__btn'
         variant='outlined'
         size='medium'
-        disabled={!pointer | !plaintext}
```
[the remark this answers is not in the document]
Comment (sublimator) [reply src/pages/exclusive-content/components/generate-form.js:90]: Was casting to a 0 or 1 ?

**Pull request merged**